crime-evidence/build.gradle

```diff
@@ -17,6 +17,7 @@ jacoco {
 def versions = [
         pitest                : "1.16.0",
         crimeCommonsClasses   : "3.4.0",
+        commonsModSchemas     : "1.5.0",
```

Comment:
Now that we are using common schemas, we can remove jsonSchema2Pojo from this build.gradle.

Reply:
Ah yeh good point, I'll remove it now 👍